src/library/scala/collection/immutable/List.scala

```diff
@@ -391,6 +391,86 @@ sealed abstract class List[+A] extends AbstractSeq[A]
     }
   }
 
+  override def filter(p: A => Boolean): Self = filterImpl(p, isFlipped = false)
```

Comment:
Did you intend to also override `filterNot`?

Reply:
I think it is on line 396